specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/backuprestore.json

```diff
@@ -33,6 +33,10 @@
           "FullBackup"
         ],
         "operationId": "FullBackup",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
```

Comment:
This is the first time I'm seeing the application of LRO annotation in a data-plane swagger. This annotation usually says that the applied operation follows the resource-manager LRO protocol. Here is the doc reference to this [annotation](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation) and the mgmt SDKs have a special code path for LRO methods. Afaik the data-plane SDKs don't do anything automatically if it is produced from a swagger with this annotation. 

Looping in Johan @johanste from the API review board to comment on whether this annotation is recommended.

Reply:
Data plane APIs also support LROs. We should verify that the generated code works (which relies on the fact that the service actually send an azure-async-operation header), but assuming that it does, the annotation should be there...

Reply:
@johanste thanks! I checked .net &java vanilla code-gen, while those generate REST API method level for LRO, there is no specific logic for polling in azure-core hence Poller won't be "generated" as a return type. The data-plane SDK's for .net & java returns Poller from the convenience layer by wrapping the auto-gened REST API LRO method. This extension is considered only when SDK's are generated for ARM.


 @vasanthrajams could you add 200 response code also under responses, like:

```
"200": {
    "description": "OK"
},
```

this is to unblock the linter, though SDKs won't use them.

Reply:
Sync-ed with Vasanta. The same pattern POST + 200 + azure-async-operation is already discussed with the team as part of another [PR](https://github.com/Azure/azure-rest-api-specs/pull/9647), the same linter error, in this case, is identified as a bug, hence we're good on this PR as well.